src/OrbitCore/Path.cpp

```diff
@@ -60,13 +69,22 @@ std::filesystem::path Path::CreateOrGetOrbitAppDataDir() {
   return path;
 }
 
-std::filesystem::path Path::GetLogFilePathAndCreateDir() {
+std::filesystem::path Path::CreateOrGetLogDir() {
   std::filesystem::path logs_dir;
   if (!absl::GetFlag(FLAGS_log_dir).empty()) {
     logs_dir = absl::GetFlag(FLAGS_log_dir);
   } else {
     logs_dir = Path::CreateOrGetOrbitAppDataDir() / "logs";
   }
   std::filesystem::create_directory(logs_dir);
-  return logs_dir / "Orbit.log";
+  return logs_dir;
+}
+
+std::filesystem::path Path::GetLogFilePath() {
+  std::filesystem::path logs_dir = CreateOrGetLogDir();
+  time_t timestamp = std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
+  std::string filename =
+      absl::StrCat("Orbit", kLogFileNameDelimiter, orbit_core::FormatTime(timestamp),
```

Comment:
Did you consider `absl::FormatTime()` instead of `orbit_core::FormatTime()`? The orbit_core thing looks like some fairly old code, I strongly feel like we should not reuse that if possible. 
If you use absl, there is also the function `absl::ParseTime()` which might simplify the `GetLogFileLifetimeMinutes`. I write a comment there as well

Reply:
Thanks for telling me about this! I changed to use `absl::FormatTime` and `absl::ParseTime` to simplify the code.